### PR TITLE
changes to remote

### PR DESCRIPTION
**Pull request by matthiasreisser**
Did a few changes to the remote capabilities

### CODE REVIEW

**Comment by matthiasreisser** [comment]
Branch can be closed after merge

File: artemis/remote/nanny.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    '''
+    Manages child processes. This class manages the start, live and deconstruction of child processes across different machines.
+    '''
+    def __init__(self):
+        self.managed_child_processes = {}
+        self.stdout_threads = {}
+        # self.original_sigint_handler = signal.getsignal(signal.SIGINT)
+        # self.original_sigterm_handler = signal.getsignal(signal.SIGTERM)
+        # signal.signal(signal.SIGINT, lambda signum, stack :self.deconstruct(signal.SIGINT,system_signal=True))
+        # signal.signal(signal.SIGTERM, lambda signum, stack :self.deconstruct(signal.SIGTERM,system_signal=True))
```
Comment (petered) [inline artemis/remote/nanny.py]: lets remove commented code unless kept for a reason, in which case explain reason

**Comment by matthiasreisser** [comment]
Removed obsolete commented out lines

**Comment by matthiasreisser** [comment]
Master merged, ready for review

File: artemis/remote/plotting/plotting_server.py [hunk 2]
```diff
@@ -45,26 +50,28 @@ def write_port_to_file(port):
     atexit.register(remove_port_file)
     port_file_path = get_local_path("tmp/plot_server/port.info", make_local_dir=True)
     if os.path.exists(port_file_path):
```
Comment (petered) [inline artemis/remote/plotting/plotting_server.py]: remove this thing if not longer active

File: artemis/remote/plotting/plotting_server.py [hunk 3]
```diff
@@ -137,105 +155,20 @@ def run_plotting_server(address, port):
...
-    plots away.
-    :param connection:
-    :param client_address:
-    :param input_queue:
-    :return:
-    """
-    while True:
-        recv_message = recv_size(connection)
-        input_queue.put(ClientMessage(recv_message, client_address))
-        # input_queue.put({"plot":recv_message,"client":client_address}, block=False)
```
Comment (petered) [inline artemis/remote/plotting/plotting_server.py]: rm commented code

**Pull request merged**